src/libraries/System.Text.Json/src/System/Text/Json/Serialization/Converters/Collection/IEnumerableDefaultConverter.cs

```diff
@@ -9,7 +9,8 @@ namespace System.Text.Json.Serialization.Converters
     /// <summary>
     /// Default base class implementation of <cref>JsonIEnumerableConverter{TCollection, TElement}</cref>.
     /// </summary>
-    internal abstract class JsonIEnumerableDefaultConverter<TCollection, TElement> : JsonCollectionConverter<TCollection, TElement>
+    internal abstract class IEnumerableDefaultConverter<TCollection, TElement>
+        : JsonCollectionConverter<TCollection, TElement>
```

Comment:
`JsonConllectionConverter`, `JsonDictionaryConverter` and `JsonObjectConverter` are not renamed becuase those are the ones that are going to become public, right?

Reply:
Correct. If that's not the case then we can revisit the naming at that point.